emperor/support_files/js/view.js

```diff
@@ -353,6 +378,130 @@ DecompositionView.prototype._fastInit = function() {
   this.markers.push(cloud);
 };
 
+/**
+ * Parallel plots closely mirroring the shader enabled _fastInit calls
+ */
+DecompositionView.prototype._fastInitParallelPlot = function()
+{
+  var positions, colors, opacities, visibilities, geometry, cloud;
+
+  // We're really just drawing a bunch of line strips...
```

Comment:
Can you remove these two lines? It is my understanding that we definitely need shaders, right? Otherwise the additional required overhead with THREE.js objects would make large plots way too slow.